### DISCUSSION TITLE
Sorry, it seems that there are no available payment methods for your state. Please contact us if you require assistance or wish to make alternate arrangements.

### DISCUSSION THREAD

**Discussion by shekman** [woocommerce/woocommerce#35461] [WooCommerce HPOS] [answered]
hello all. I understand this is a common thread but I'm the latest victim of this annoying message. I've deactivated/reactivated the plugins, checked all theme, wordpress, php and everything is still up to date but getting the message. Any tips????
https://fts.pgi.mybluehost.me/product/test-2/ 

![image](https://user-images.githubusercontent.com/3935983/199355437-a1c9fd74-f1ad-46f7-965c-384c7f4f010c.png)

**Comment by barryhughes** [comment, marked as answer]
Hi @shekman. You've landed in the discussion area for the [WooCommerce HPOS (High-Performance Order Storage) project](https://github.com/woocommerce/woocommerce/discussions/categories/woocommerce-hpos) ... it doesn't sound like your question relates to that project and so it would be better to [ask this elsewhere](https://woocommerce.com/document/how-to-get-help/) or even [post a bug report directly](https://github.com/woocommerce/woocommerce/issues/new?assignees=&labels=&template=1-bug-report.yml) if you are confident this is a bug. 

Of course, if we have misunderstood and this does relate to HPOS please provide some additional context.

Thanks!
